### PR TITLE
Remove libandroidfw: it was never used

### PR DESCRIPTION
**Pull request by eli-schwartz**
I actually cannot find any traces of it *ever* being used but it certainly isn't used now. It builds a bunch of code for a static library that is never consumed, which is both slow and also potentially erroring out (since the unused code contains strict-aliasing violations).

Fixes: #148

### CODE REVIEW

File: vendor/CMakeLists.txt [hunk 1]
```diff
@@ -44,7 +44,6 @@ add_compile_definitions(_FILE_OFFSET_BITS=64 _LARGEFILE64_SOURCE)
 set(android-vendored
 	avb
 	adb
-	base
```
Comment (anatol) [inline vendor/CMakeLists.txt:47]: I don't think you want to remove `base` submodule here.

**Review by anatol — CHANGES_REQUESTED** [review]

File: vendor/CMakeLists.txt (hunk 1 above)
Comment (anatol) [reply vendor/CMakeLists.txt:47]: Oh wait, does it mean that `libandroidfw` was the only library from `base` used?

Comment (eli-schwartz) [reply vendor/CMakeLists.txt:47]: Correct, that submodule existed solely in order to provide libandroidfw -- I originally tested this via `rm -rf vendor/base` and everything still built.

**Comment by eli-schwartz** [comment]
CI failure is the return of the terrible horrible no good very bad GitHub Actions Python...

**Review by anatol — APPROVED** [review]

**Comment by anatol** [comment]
thank you @eli-schwartz for your contribution!

**Comment by eli-schwartz** [comment]
I actually realized I should have deleted the patches/base/ as well, and pushed an amended commit approximately 90 seconds after you merged the PR. :D

**Comment by anatol** [comment]
> I actually realized I should have deleted the patches/base/ as well, and pushed an amended commit approximately 90 seconds after you merged the PR. :D

np, please send another PR.

**Comment by eli-schwartz** [comment]
https://github.com/nmeum/android-tools/pull/150

**Pull request merged**